packages/client/base/src/types/index.ts

```diff
@@ -1,10 +1,9 @@
-export * from "./blockchain";
+// export * from "@crossmint/common-sdk-base"
```

Comment:
can we remove instead of commenting out?
```suggestion
```

Reply:
yep, missed it